javascript-sdk/examples/interact-with-external-or-existing-browser/README.md

```diff
@@ -0,0 +1,56 @@
+# Example script: login into websites
+
+This is an example shows how to login into websites by retrieving and interacting with web elements in AgentQL.
+
+## Run the script
+
+- [Install AgentQL SDK](https://docs.agentql.com/javascript-sdk/installation)
+- Save this javascript file locally as **log-into-sites.js**
+- Run the following command from the project's folder:
+
+```bash
+node log-into-sites.js
+```
+
+## Play with the query
+
+Install the [AgentQL Debugger Chrome extension](https://docs.agentql.com/installation/chrome-extension-installation) to play with the AgentQL query. [Learn more about the AgentQL query language](https://docs.agentql.com/agentql-query/query-intro)
+
+# Example script: interact with external or existing browser
```

Comment:
You have 2 blocks with 2 different scripts described. Pls, fix it.

The second bock has no links in markdown, seems like you copied plain text only